sonar-application/src/main/java/org/sonar/application/App.java

```diff
@@ -176,6 +176,10 @@ public static void main(String[] args) throws InterruptedException {
     AppLogging logging = new AppLogging();
     logging.configure(props);
 
+    ClusterProperties clusterProperties = new ClusterProperties(props);
+    clusterProperties.validate();
+    Cluster cluster = new Cluster(clusterProperties);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "cluster". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "cluster" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)